sdk/communication/azure-communication-common/README.md

```diff
@@ -3,7 +3,7 @@
 This package contains common code for Azure Communication Service libraries.
 
 [Source code](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/communication/azure-communication-common)
-| [API reference documentation](https://azure.github.io/azure-sdk-for-android/sdk/communication/azure-communication-common/azure-communication-common/index.html)
+| [API reference documentation](https://azure.github.io/azure-sdk-for-android/azure-communication-common/index.html)
```

Comment:
@vcolin7 it was my bad to have GitHub io with incorrect folder structure, I think we should fix dir-structure to follow the standard instead, I'll fix GitHub.io.

Reply:
I mean we don't have to fix README to address the wrong doc structure.

Reply:
I'll close this PR for now then :)